src/Task/CopyDevPaths.php

```diff
@@ -124,12 +140,56 @@ private function copyCustomPath(string $pathConfigKey, Io $io): int
             return 0;
         }
 
+        return $this->copy($sourcePaths, $target, $io);
+    }
+
+    /**
+     * @param Io $io
+     *
+     * @return int
+     */
+    private function copyMuPluginsPackages(Io $io): int
+    {
+        $rootPackage = $this->composer->getPackage();
+        $repositoryManager = $this->composer->getRepositoryManager();
+        $target = $this->directories->muPluginsDir();
+
         $errors = 0;
+        foreach ($rootPackage->getRequires() as $link) {
+            /** @var ConstraintInterface $constraint */
+            $constraint = $link->getConstraint();
+            if (!$constraint instanceof Constraint) {
+                continue;
+            }
+            $package = $repositoryManager->findPackage($link->getTarget(), $constraint);
+            if ($package && $package->getType() === Config::PACKAGE_TYPE_MULTIPLE_MU_PLUGINS) {
+                $vendorDir = $this->composer->getConfig()->get('vendor-dir');
+                $packagePath = $vendorDir . '/' . $package->getPrettyName();
+                $packageFiles = Finder::create()->in($packagePath)->files()->name('*.php');
+
+                $errors += $this->copy($packageFiles, $target, $io, true);
+            }
+        }
+
+        return $errors;
+    }
 
+    /**
+     * @param Finder $sourcePaths
+     * @param string $target
+     * @param Io $io
+     * @param bool $singleFiles If to copy the whole folder or single files inside it
+     *
+     * @return int The number of errors encountered. 0 if all the files have been copied correctly.
+     */
+    private function copy(Finder $sourcePaths, string $target, Io $io, bool $singleFiles = false): int
+    {
+        $errors = 0;
         /** @var SplFileInfo $sourcePathInfo */
         foreach ($sourcePaths as $sourcePathInfo) {
             $sourcePath = $sourcePathInfo->getPathname();
-            $targetPath = "{$target}/" . $sourcePathInfo->getBasename();
+            $finalTarget = $singleFiles ? $sourcePathInfo->getFilename() : $sourcePathInfo->getBasename();
+            $targetPath = "{$target}/{$finalTarget}";
```

Comment:
Why do you need to deal differently for folder/single files?

Even because I think that `$sourcePathInfo->getPathname()` and `$sourcePathInfo->getFilename()`  would return exact sdame value, but the latter would work _only_ for files, while `getPathname` would work _also_ for files.